frontend/src/styles/global.css

```diff
@@ -0,0 +1,11 @@
+body {
+  padding: 30px;
+  font-family: 'Poppins', 'Roboto', sans-serif;
+  background-color: #f9f9f9;
+  color: #333;
```

Comment:
Avoid using hard code. There is a good way in CSS to define constant colors in CSS or any constant. You can try to learn about it, it is very powerful.